docs/index.md

```diff
@@ -39,7 +39,7 @@ For this analysis, the team chose counties with similar population demographics
 
 ## Vaccine Effectiveness - County Comparison
 
-The visulization indicates a very noticable difference between the number of deaths in a county with high vaccination rates (Kent, RI) and the number of deaths in a county with low vaccination rates (Tuscaloosa, AL). It is important to note that this analysis only compares two specific counties and does not take into consideration other relevant factors such as mask wearing and social distancing. However, it is still interesting to see that the conclusion from the analysis of all U.S. counties is the same as the analysis from individual county perspective. 
+The visualization indicates a very noticeable difference between the number of deaths in a county with high vaccination rates (Kent, RI) and the number of deaths in a county with low vaccination rates (Tuscaloosa, AL). It is important to note that this analysis only compares two specific counties and does not take into consideration other relevant factors such as mask wearing and social distancing. However, it is still interesting to see that the conclusion from the analysis of all U.S. counties is the same as the analysis from individual county perspective. 
 
 ## About the Data
```

Comment:
We mention this same information in the beginning overview. Can we just include a hyperlink to the data sources in the overview and remove this section altogether?

Reply:
Is this comment referring to the "about the data" again?